src/Compilers/CSharp/Portable/Binder/Binder_InterpolatedString.cs

```diff
@@ -92,20 +98,50 @@ private BoundExpression BindInterpolatedString(InterpolatedStringExpressionSynta
                             }
 
                             builder.Add(new BoundStringInsert(interpolation, value, alignment, format, null));
+                            if (value.ConstantValue != null)
+                            {
+                                if (!value.ConstantValue.IsString || value.ConstantValue.IsBad || value.ConstantValue.IsNull)
```

Comment:
Why is `null` special cased here as a non-constant? That is a legal `string` constant value. Please add a PROTOTYPE comment to follow up on that.

Reply:
`(ConstantValue)null != ConstantValue.Null`

---
In reply to: [447269087](https://github.com/dotnet/roslyn/pull/45396#discussion_r447269087) [](ancestors = 447269087)

Reply:
If we're talking about value.ConstantValue, that was null checked to prevent a whole host of NullPointerExceptions. If we're taking about the extra case in the if statement beneath it, I'd honestly forgotten if it was merely out of an abundance of caution or not. Removed it in commit 16, lets see if it works.

---
In reply to: [447269087](https://github.com/dotnet/roslyn/pull/45396#discussion_r447269087) [](ancestors = 447269087)